src/App.js

```diff
@@ -1,14 +1,33 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+import "./app.css";
+
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state = {
+      tasks: []
+    };
+  }
+  
+  addTodo = (task) => {
+    const newTask = {
+      id: Date.now(),
```

Comment:
I would not recommend using Data.now(), I'd recommend finding a way to increment your id when it is being generated. For now this is ok but you'll have to think of ways to replace this down the line.